Cargo.lock

```diff
@@ -3062,11 +3051,6 @@ name = "smallbitvec"
 version = "1.0.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
-[[package]]
```

Comment:
This is deduplicating smallvec. It should be removed from the `[ignore]packages` list of servo-tidy

Reply:
Oops, thanks. Fixed that and import order to make tidy happy.